app/helpers/conferences_helper.rb

```diff
@@ -1,9 +1,9 @@
 module ConferencesHelper
-  GLYPHS = %w(list fire leaf music user asterisk plus minus cloud envelope pencil glass search heart 
-    star star-empty th th-large th-list ok remove cog trash home file time road download 
-    download-alt upload inbox repeat refresh lock qrcode barcode tag tags bookmark picture 
-    map-marker move play stop plus-sign minus-sign remove-sign ok-sign question-sign info-sign 
-    remove-circle ok-circle ban-circle gift eye-open calendar random comment magnet hdd bell 
+  GLYPHS = %w(list fire leaf music user asterisk plus minus cloud envelope pencil glass search heart
+    star star-empty th th-large th-list ok remove cog trash home file time road download
+    download-alt upload inbox repeat refresh lock qrcode barcode tag tags bookmark picture
+    map-marker move play stop plus-sign minus-sign remove-sign ok-sign question-sign info-sign
+    remove-circle ok-circle ban-circle gift eye-open calendar random comment magnet hdd bell
     bullhorn)
 
   def scheduleColor(color)
```

Comment:
this method is totally not readable. Can be refactored with `inject` + normal variable names